t04_refactored.py

```diff
@@ -410,8 +409,98 @@ def team_11_adv():
 
 
 def team_12_adv():
-    pass
-    # TODO Add your code here
+    # TEAM 12
+    # Kevin and Nathan
```

Comment:
These lines belong inside the docstring (or it breaks PyCharm's ability to parse the docstring)